manifests/manifest-prod.yml

```diff
@@ -0,0 +1,58 @@
+---
+defaults: &defaults
+  instances: 1
+  stack: cflinuxfs4
+  buildpacks:
+    - python_buildpack
+  memory: 1G
+  services:
+    - fecfile-api-rds
+    - fecfile-api-s3
+    - fecfile-api-redis
+    - fecfile-api-creds-prod
+  env:
+    DISABLE_COLLECTSTATIC: 1
+    DJANGO_SETTINGS_MODULE: fecfiler.settings.production
+    FFAPI_COOKIE_DOMAIN: fecfile.fec.gov
+    LOGIN_REDIRECT_CLIENT_URL: https://fecfile.fec.gov
+    LOGIN_REDIRECT_SERVER_URL: https://api.fecfile.fec.gov/api/v1/oidc/login-redirect
+    LOGOUT_REDIRECT_URL: https://api.fecfile.fec.gov/api/v1/oidc/logout-redirect
+    PRODUCTION_OPEN_FEC_API: https://api.open.fec.gov/v1/
+    STAGE_OPEN_FEC_API: https://api-stage.open.fec.gov/v1/
+    FEC_FILING_API: https://efoservices.stage.efo.fec.gov
+    SESSION_COOKIE_AGE: 64800
+    BP_PIP_VERSION: latest
+    LOG_FORMAT: KEY_VALUE
+    SYSTEM_STATUS_CACHE_AGE: 60
+    INITIAL_POLLING_INTERVAL: 10
+    INITIAL_POLLING_DURATION: 900
+    SECONDARY_POLLING_INTERVAL: 3600
+    SECONDARY_POLLING_DURATION: 86400
+    # ---- FEATURE FLAGS ----
+    FLAG__COMMITTEE_DATA_SOURCE: PRODUCTION # Values are PRODUCTION, TEST, and MOCKED
+    ENABLE_DEVELOPER_COMMANDS: False
```

Comment:
Let's add FEC_AGENCY_ID: FEC from the old web-services environment varaibles.  All the other environment variables are good